pkg/component/grpc_test.go

```diff
@@ -71,6 +71,15 @@ func (gs *gsImplementation) GetGatewayConnectionStats(ctx context.Context, _ *tt
 	return &ttnpb.GatewayConnectionStats{}, nil
 }
 
+func (*gsImplementation) BatchGetGatewayConnectionStats(ctx context.Context,
+	_ *ttnpb.BatchGetGatewayConnectionStatsRequest,
+) (*ttnpb.BatchGetGatewayConnectionStatsResponse, error) {
+	if err := clusterauth.Authorized(ctx); err != nil {
+		return nil, err
+	}
```

Comment:
Why test with cluster auth?

Reply:
This is a copy of https://github.com/TheThingsNetwork/lorawan-stack/blob/34b15a6e0e3e7a3735707e56c971b775fbc1542d/pkg/component/grpc_test.go#L67-L72 
I'm satisfying the interface. I didn't implement this so I don't have the context.